doc/advanced.rst

```diff
@@ -267,6 +267,23 @@ arguments, but after the environment and the context. For instance, a call to
 ``'foo'|a_path_b()`` will result in the following arguments to be passed to
 the filter: ``('a', 'b', 'foo')``.
 
+Deprecated Filters
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.21
+    Support for deprecated filters was added in Twig 1.21.
```

Comment:
Not sure if this is a problem, but normally you have an empty line after the block start.
*From what I know the first contains options like `:lineno:` but I'm not sure if Sphixdoc will accept tex directly after the block start :)*

```
.. versionadded:: 1.21

    Support for deprecated filters was added in Twig 1.21.
```